### PR TITLE
docs: define the update.conf

### PR DESCRIPTION
**Pull request by vbatts**
Closes https://github.com/kinvolk/Flatcar/issues/224

Signed-off-by: Vincent Batts <vbatts@kinvolk.io>

### CODE REVIEW

File: docs/clusters/management/update-conf.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+description: >
+  Definitions for Flatcar update configuration.
+weight: 100
+---
+
+Flatcar Container Linux uses [`update_engine`][update_engine] to check and fetch new updates from the [Nebraska Update Service](https://github.com/kinvolk/nebraska).
+
+## Location
+
+The client-side configuration these updates is stored in `/etc/flatcar/update.conf`.
```
Comment (invidian) [inline docs/clusters/management/update-conf.md:13]:
```suggestion
The client-side configuration of these updates is stored in `/etc/flatcar/update.conf`.
```

File: docs/clusters/management/update-conf.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    * `FLATCAR_RELEASE_BOARD=`
+    * `FLATCAR_RELEASE_APPID=`
+
+## Fields
+
+Default installs of Flatcar will likely only have `GROUP=` set.
+
+* `GROUP`
+  * The channel/group this host will pull updates from
+  * public channels will be: `stable`, `beta`, `alpha`
```
Comment (invidian) [inline docs/clusters/management/update-conf.md:37]: What about `edge`?

**Review by invidian — COMMENTED** [review]
Just some nits/suggestions, otherwise looking good!

File: docs/clusters/management/update-conf.md (hunk 2 above)
Comment (vbatts) [reply docs/clusters/management/update-conf.md:37]: we'll be renaming (or hopefully removing) edge. These three are the long established ones.

**Comment by vbatts** [comment]
updated PTAL

**Review by invidian — APPROVED** [review]
LGTM. Maybe a bit minimalistic, but clear and concise.

File: docs/clusters/management/update-conf.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+* `/usr/share/flatcar/release`
+  * Generated at build time of the image/payload build
+  * will typically contain:
+    * `FLATCAR_RELEASE_VERSION=`
+    * `FLATCAR_RELEASE_BOARD=`
+    * `FLATCAR_RELEASE_APPID=`
+
+## Fields
+
+Default installs of Flatcar will likely only have `GROUP=` set.
```
Comment (pothos) [inline docs/clusters/management/update-conf.md:33]: This sentence doesn't sound right. All have to be set but the defaults can be used.

I would reword it as:
```suggestion
Default installs of Flatcar will likely not need custom settings, and an empty `/etc/flatcar/update.conf` is sufficient.
```

File: docs/clusters/management/update-conf.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+* `GROUP`
+  * The channel/group this host will pull updates from
+  * public channels will be: `stable`, `beta`, `alpha`
+  * otherwise these are UUIDs
+* `SERVER`
+  * The update server to reach for updates
+  * default community server is: https://public.update.flatcar-linux.net/v1/update/
+* `FLATCAR_RELEASE_VERSION`
+  * The current version of this machine's version
```
Comment (pothos) [inline docs/clusters/management/update-conf.md:43]:
```suggestion
  * The current version of this machine
```

**Comment by vbatts** [comment]
@joaquimrocha with the new docs, is there special linking to make this doc discoverable?

**Comment by pothos** [comment]
Added it to the index
